EIPS/eip-1238.md

```diff
@@ -0,0 +1,152 @@
+---
+eip: 1238
+title: Non-Transferrable Non-Fungible Tokens (Soulbound Tokens)
+author: Nicola Greco, Tim Daubenschütz <tim@daubenschuetz.de>
```

Comment:
```suggestion
author: Nicola Greco, Tim Daubenschütz (TimDaub), Tim Daubenschütz <tim@daubenschuetz.de>
```
```suggestion
author: Nicola Greco, Tim Daubenschütz (TimDaub)
```
At least one author needs a GitHub name.  Either of the above are acceptable.  I'm not sure if the bot will allow an author with neither GitHub nor email...